examples/scripts/ppo/ppo.py

```diff
@@ -86,7 +88,7 @@
     ################
     # Dataset
     ################
-    dataset = load_dataset("trl-internal-testing/descriptiveness-sentiment-trl-style", split="descriptiveness")
+    dataset = load_dataset(script_args.dataset_name, split="descriptiveness")
```

Comment:
We could use split=script_args.dataset_train_split here so the user can run with another dataset?

Reply:
Done in 747fb7ca9ae148844b30f18c8a5136f53ce8fac8